### DISCUSSION TITLE
ESRGAN Folder not here

### DISCUSSION THREAD

**Discussion by BenDes21** [lllyasviel/stable-diffusion-webui-forge#61] [General]
Hi, Im looking to install new upscalers for Hires Fix but the folder ESRGAN ( "..\Stable diffusion\stable-diffusion-webui\models\ESRGAN" ) from the original a1111 is not here :( Is it normal ? Thanks

**Comment by bendecruz745** [comment]
Yes it's normal, I just created the folder myself and put the upscaler file in there and it functions fine.

**Reply by BenDes21** [reply]
Thanks you

**Reply by x999xmot** [reply]
also the first time you that you use ESRGAN, it will create a folder for you.
